modules/mod_audio_fork/lws_glue.cpp

```
@@ -211,6 +212,7 @@ namespace {
             break;
             case AudioPipe::CONNECTION_DROPPED:
               // first thing: we can no longer access the AudioPipe
+              fork_session_cleanup(session, "stop", 0)
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If fork_session_cleanup is a macro then please configure it.

(unknownMacro)